internal/trace/trace.go

```diff
@@ -32,16 +32,15 @@ import (
 )
 
 const (
-	telemetryPlatformTracingOpenCensus    = "opencensus"
-	telemetryPlatformTracingOpenTelemetry = "opentelemetry"
-	telemetryPlatformTracingVar           = "GOOGLE_API_GO_EXPERIMENTAL_TELEMETRY_PLATFORM_TRACING"
+	TelemetryPlatformTracingOpenCensus    = "opencensus"
+	TelemetryPlatformTracingOpenTelemetry = "opentelemetry"
+	TelemetryPlatformTracingVar           = "GOOGLE_API_GO_EXPERIMENTAL_TELEMETRY_PLATFORM_TRACING"
+	OpenTelemetryTracerName               = "cloud.google.com/go"
 )
 
 var (
-	// TODO(chrisdsmith): Should the name of the OpenTelemetry tracer be public and mutable?
-	openTelemetryTracerName     string = "cloud.google.com/go"
-	openTelemetryTracingEnabled bool   = strings.EqualFold(strings.TrimSpace(
-		os.Getenv(telemetryPlatformTracingVar)), telemetryPlatformTracingOpenTelemetry)
+	OpenTelemetryTracingEnabled bool = strings.EqualFold(strings.TrimSpace(
```

Comment:
Should we be adding a comment for `OpenTelemetryTracingEnabled` saying that this field should be modified only for unit tests by go client libraries and should not be modified by any other applications?

Reply:
Sure, good idea.

Reply:
Done, thanks.